### ISSUE TITLE
Filler words detection

### ISSUE THREAD

**Issue by magrenimish** [Shahabks/my-voice-analysis#29]
How are filler words being detected? 
The function mysp.mysppaus(p,c) seems to only detect the pauses.

**Comment by kalpesh-maru** [comment]
Yes, it says fillers and pauses but their counts should be given separately, right now cannot say this can provide just filler word count
